fulp_modules/features/mentors/follow.dm

```diff
@@ -22,3 +27,8 @@
 	to_chat(GLOB.admins, span_adminooc("<span class='prefix'>MENTOR:</span> <EM>[key_name(usr)]</EM> is no longer following <EM>[key_name(mentor_datum.following)]</span>"))
 	log_mentor("[key_name(usr)] stopped following [key_name(mentor_datum.following)]")
 	mentor_datum.following = null
+	//Add back mentor's blindness/color blindness
+	usr.reload_fullscreen()
```

Comment:
If you are cured of blindness between this do you keep it lol

Reply:
I’ve never looked into this stuff much but it looks dangerous

Reply:
> If you are cured of blindness between this do you keep it lol

Yes

Reply:
Doesn't it make this not good? If I get my eyes stabbed out while mfollowing someone then I am permablind and would need admins to know what caused it and know how to fix it

Reply:
I miss understood. No you would be fine. This is not true for color blindness trying to fix it now.